source/index.rst

```diff
@@ -18,6 +18,30 @@ With `Inrupt PodBrowser`_, you can
 
 - Manage access controls to your data.
 
+Browser Support
```

Comment:
Personally, I would expand the list of supported browser to those shown here: https://inrupt.atlassian.net/browse/SOLIDOS-355?focusedCommentId=13359. We need to support Samsung on Android and we cannot expect users to be on the very latest versions. But it really is Product's call.

Reply:
Will check with Oz.  I'm fine with stating we support Chrome on Android and Safari/Chrome on iPhones - but at least from Virginia's comments, it doesn't seem as though we tested with Samsung Internet.

Reply:
We have access to BrowserStack so we should be able to test on Samsung devices.

Reply:
Updated to state the latest 2 versions instead of giving a baseline since we'd be out of sync as soon as they release others.
Also, using the guidance per public documentation repo.